CHANGELOG.md

```diff
@@ -1,25 +1,17 @@
-# NEXT RELEASE
+# 2.8.0 Release notes
 
 ### Bugfixes
 
 * Fix a race condition in encrypted files which can lead to
   crashes on devices using OpenSSL (Android).
   PR [#2616](https://github.com/realm/realm-core/pull/2616).
 
-### Breaking changes
-
-* Lorem ipsum.
-
 ### Enhancements
 
 * Enable encryption on watchOS.
   Cocoa issue [#2876](https://github.com/realm/realm-cocoa/issues/2876).
-
------------
-
-### Internals
```

Comment:
There are no internal changes, I don't think we keep a section if it is completely empty?